client/src/components/PreviewPopUp.tsx

```diff
@@ -20,7 +20,7 @@ export default function PreviewPopUp(props: {
   eventCapacity: string;
   location: string;
   openPopup: boolean;
-  handleClose: any;
+  handleClose: () => any;
```

Comment:
```suggestion
  handleClose: () => void;
```

Does the function actually return `any`?

Reply:
my bad, I meant void